conda/recipes/kvikio/meta.yaml

```diff
@@ -32,6 +32,9 @@ build:
     - SCCACHE_S3_KEY_PREFIX=kvikio-linux64 # [linux64]
     - SCCACHE_S3_USE_SSL
     - SCCACHE_S3_NO_CREDENTIALS
+  script:
+    - cd python/kvikio
+    - python -m pip install . -vv
```

Comment:
I think it'd be better to follow the patterns other RAPIDS repos use, which is like:

1. have a `build.sh` in the recipe directory
2. omit `script:` block from the `build:` section of conda recipe
3. have that recipe run the `build.sh` from the root of the repo
4. keep all logic about directory structure, `pip` arguments, etc. in root-level `build.sh`

See `raft` as an example:

https://github.com/rapidsai/raft/blob/branch-24.08/conda/recipes/pylibraft/build.sh